openmls/src/key_store.rs

```diff
@@ -1,53 +1,56 @@
 //! Serialization for key store objects.
 
 use crate::{
-    credentials::CredentialBundle, key_packages::KeyPackageBundle, schedule::psk::PskBundle,
+    credentials::CredentialBundle, key_packages::KeyPackageBundle, prelude::LibraryError,
+    schedule::psk::PskBundle,
 };
 
 use openmls_traits::key_store::{FromKeyStoreValue, ToKeyStoreValue};
 
 // === OpenMLS Key Store Types
 
 impl FromKeyStoreValue for KeyPackageBundle {
-    type Error = &'static str;
+    type Error = LibraryError;
```

Comment:
I am not sure whether reusing `LibraryError` here is a good idea. Alternatively, I could define four structs to be more specific. It will be more verbose though.

Reply:
I think `LibraryError` is fine here for now. There's nothing to recover from the error information.